docs/src/index.md

```diff
@@ -0,0 +1,68 @@
+# Pkg3.jl
+
+!!! warning
+    This documentation is a work in progress and the information in it might be or become outdated.
+
+Sections:
+
+```@contents
+Pages = [
+    "index.md"]
+```
+
+## Introduction
+
+Pkg3 is the package manager for Julia.
+
+## Getting Started
+
+The Pkg REPL-mode is entered using from the Julia REPL using the key `]`. To start a new project, use `pkg> new`.
+
+```
+pkg> new HelloWorld
+```
+
+This creates a new package `HelloWorld` with the following files
```

Comment:
I think we should design this to generate projects in general – both packages (reusable) and applications (standalone). From my wip glossary:

> **Project:** a source tree with a standard layout for a Julia project, including a `src` directory for the main body of Julia source code, a `test` directory for testing the project, `docs` for documentation files, and optionally `build` for build scripts and outputs.
> 
> **Application:** a project which provides standalone functionality that is not intended to be reused other Julia projects.
> 
> **Package:** a project which provides functionality that can be used by other Julia projects via `import X` or `using X`. A package will typically have a project file with a `uuid` entry giving its package UUID. This UUID is used to identify the package when other projects depend on it.

Most of the infrastructure is common, but this is a good opportunity to support both use cases with a single shared body of development tools.

Reply:
Ok so `--application` or `--package` options? And an `application` does not get a UUID?

Reply:
No harm in generating one, I guess. Then if someone decides to convert their application to a package (which is pretty common), they're good to go. I think the main difference is actually that an application can set global configuration while a package cannot since it will be loaded by something else which controls global configuration.